src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,12 +1150,12 @@ private static String getUsageInfoForExitCommand() {
     /**
      * Removes sign(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param fullString  Parameter as a string
```

Comment:
It is good that you changed the comments about the params. 

But for the new abstract method is the previous header comment still valid (line 1151)?